src/components/CardSeller/index.js

```diff
@@ -0,0 +1,29 @@
+import { Avatar } from '../Avatar';
+import { Tags } from '../Tags';
+import { Wrapper, CategoriesWrapper, MarketInfos, MarketWrapper, SellerAvatar } from './styles';
+
+export const CardSeller = ({ seller }) => {
+  return (
+    <Wrapper>
+      <SellerAvatar>
+        <Avatar size="small" backgroundColor={({ theme }) => theme.palettes.primaryGreen.v4} />
```

Comment:
Isso aqui não é muito interessante.

Utilize o hook useTheme do styled-componentes.

```jsx
import { useTheme } from 'styled-components'

const theme = useTheme()
```

```suggestion
        <Avatar size="small" backgroundColor={theme.palettes.primaryGreen.v4} />
```